pkg/catalog/catalog_test.go

```diff
@@ -314,8 +313,9 @@ func MockImageStream() *imagev1.ImageStream {
 	tags := map[string]string{
 		"12": "docker.io/rhscl/nodejs-12-rhel7:latest",
 		"10": "docker.io/rhscl/nodejs-10-rhel7:latest",
-		"8":  "docker.io/rhoar-nodejs/nodejs-8:latest",
+
 		// an unspported one
+		"8": "docker.io/rhoar-nodejs/nodejs-8:latest",
 		"6": "docker.io/rhoar-nodejs/nodejs-6:latest",
```

Comment:
Why not remove nodejs-6 image as well?

Reply:
Kept it here just for UTs validation step on more values. 

Validation done here - https://github.com/openshift/odo/pull/3587/files#diff-1dfbb1796e8d3ca31e26fe9d003bb488R168

Reply:
oh. my bad